README.md

```diff
@@ -319,7 +305,7 @@ Add the following dependencies:
 
 All functional tests will be created in `<yourProject>/src/test/java/org/example`.
 
-_Note that until version 1.4.1 of fabric-gateway-java is published in maven central, the functional tests will use version 1.4.1-SNAPSHOT._
+**_fabric-gateway-java version 1.4.2 has been published in maven central. Please update this dependency for functional tests created with a previous release._**
```

Comment:
**_If you have previously generated Java functional tests during its BETA, please update the `fabric-gateway-java` dependency from `1.4.1-SNAPSHOT` to `1.4.2`_**

Reply:
Done.